src/services/EcoBridge/Socket/Socket.lists.ts

```diff
@@ -0,0 +1,1760 @@
+      symbol: 'kUSDC',
+      address: '0xa124ff1e97e7f3e4a796f6a2d3bf5d0e2d41973d',
+      chainId: 1
+    },
+    {
+      decimals: 18,
+      name: 'Synth sUSD',
+      symbol: 'sUSD',
+      logoURI: 'https://assets.coingecko.com/coins/images/5010/thumb/YAIS3fUh.png?1626267646',
+      address: '0xA970AF1a584579B618be4d69aD6F73459D112F95',
+      chainId: 42161
+    },
+    {
+      decimals: 18,
+      name: 'Synth sUSD',
+      symbol: 'sUSD',
+      logoURI: 'https://assets.coingecko.com/coins/images/5010/thumb/YAIS3fUh.png?1626267646',
+      address: '0x57ab1ec28d129707052df4df418d58a2d46d5f51',
+      chainId: 1
+    }
+  ],
+  '100-42161': [
+    {
+      decimals: 18,
+      name: 'Dai Stablecoin',
+      symbol: 'DAI',
+      logoURI: 'https://maticnetwork.github.io/polygon-token-assets/assets/dai.svg',
+      address: '0xda10009cbd5d07dd0cecc66161fc93d7c9000da1',
+      chainId: 42161
+    },
+    {
+      decimals: 6,
+      name: 'USD Coin Arb1',
+      symbol: 'USDC',
+      logoURI: 'https://maticnetwork.github.io/polygon-token-assets/assets/usdc.svg',
+      address: '0xddafbb505ad214d7b80b1f830fccc89b60fb7a83',
+      chainId: 100
+    },
+    {
+      decimals: 6,
+      name: 'USD Coin Arb1',
+      symbol: 'USDC',
+      logoURI: 'https://maticnetwork.github.io/polygon-token-assets/assets/usdc.svg',
+      address: '0xff970a61a04b1ca14834a43f5de4533ebddb5cc8',
+      chainId: 42161
+    },
+    {
+      decimals: 6,
+      name: 'Tether USD',
+      symbol: 'USDT',
+      logoURI: 'https://maticnetwork.github.io/polygon-token-assets/assets/usdt.svg',
+      address: '0x4ecaba5870353805a9f068101a40e0f32ed605c6',
+      chainId: 100
+    },
+    {
+      decimals: 6,
+      name: 'Tether USD',
+      symbol: 'USDT',
+      logoURI: 'https://maticnetwork.github.io/polygon-token-assets/assets/usdt.svg',
+      address: '0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9',
+      chainId: 42161
+    },
+    {
+      decimals: 8,
+      name: 'Wrapped BTC',
+      symbol: 'WBTC',
+      logoURI: 'https://maticnetwork.github.io/polygon-token-assets/assets/wbtc.svg',
+      address: '0x8e5bbbb09ed1ebde8674cda39a0c169401db4252',
+      chainId: 100
+    },
+    {
+      decimals: 8,
+      name: 'Wrapped BTC',
+      symbol: 'WBTC',
+      logoURI: 'https://maticnetwork.github.io/polygon-token-assets/assets/wbtc.svg',
+      address: '0x2f2a2543b76a4166549f7aab2e75bef0aefc5b0f',
+      chainId: 42161
+    },
+    {
+      decimals: 18,
+      name: 'Wrapped Ether',
+      symbol: 'WETH',
+      logoURI: 'https://maticnetwork.github.io/polygon-token-assets/assets/weth.svg',
+      address: '0x6a023ccd1ff6f2045c3309768ead9e68f978f6e1',
+      chainId: 100
+    },
+    {
+      decimals: 18,
+      name: 'Wrapped Ether',
+      symbol: 'WETH',
+      logoURI: 'https://maticnetwork.github.io/polygon-token-assets/assets/weth.svg',
+      address: '0x82af49447d8a07e3bd95bd0d56f35241523fbab1',
+      chainId: 42161
+    }
+  ]
+}
```

Comment:
Is there a way to quickly verify all addresses here? This is potentially risky. 

cc @0xVenky